src/test/java/br/com/gotn/devopscourse/DevOpsCourseProjectApplicationTests.java

```diff
@@ -21,15 +21,15 @@
 	private static MockMvc mockMvc;
 
 	@Autowired
-	private static WebApplicationContext context;
+	private WebApplicationContext context;
 
-	@BeforeAll
-	static void setUp() throws Exception {
+	@BeforeEach
+	void setup() {
 		mockMvc = MockMvcBuilders.webAppContextSetup(context).build();
 	}
 
 	@Test
-	public void welcomeTest() throws Exception {
+	void welcomeTest() throws Exception {
```

Comment:
cool test, here's a better method name:
```suggestion
	void welcome() throws Exception {
```